db/schema.rb

```diff
@@ -10,74 +10,74 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_11_28_192700) do
+ActiveRecord::Schema.define(version: 2019_01_18_194740) do
 
-  create_table "admin_rates", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
-    t.bigint "line_item_id"
+  create_table "admin_rates", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
```

Comment:
These changes shouldn't be included as we normally have been using BIGINTs, not INTEGERs for our ID columns for a while. I don't recall if that was done by a migration or rake task but you probably want to run that locally and remove those schema changes

Reply:
Yeah, not sure how this got in, removed.